swagger.json

```diff
@@ -1821,6 +1821,69 @@
             }
           }
         },
+        "/fleet/vehicles/stats": {
+            "parameters": [
+                {
+                  "$ref": "#/parameters/accessTokenParam"
+                },
+                {
+                  "name": "startMs",
+                  "description": "Time in Unix epoch milliseconds for the start of the query (cannot exceed 1 hour)",
+                  "required": true,
```

Comment:
Do we want to make startMs and endMs required params? For instance, we could default endMs to `now`.

Reply:
Both are required in `/fleet/vehicles/locations`, and I'd prefer to have `/stats` match that endpoint.